src/Queue.php

```diff
@@ -263,4 +267,44 @@ public static function handleGuzzleError(\GuzzleHttp\Exception\GuzzleException $
 		// other exceptions like 3xx (TooManyRedirectsException) or 4xx (\GuzzleHttp\Exception\ClientException) are unrepeatable and we want to throw exception
 		throw $guzzleException;
 	}
+
+	/**
+	 * Metoda zpracující callbacky nezpracovaných entit
+	 * bez rabbita a consumeru
+	 * @throws \Exception
+	 */
+	public function processUnprocessedEntities() {
```

Comment:
proc se to nejmenuje jen "process" ?

Reply:
tak se uz jmenuje ta metoda ktera zpracovava tu frontu z rabbita v te service -> prozatím tedy necháváme tak..